include/curves/optimization/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-SET(${PROJECT_NAME}_HELPERS_HEADERS
+SET(${PROJECT_NAME}_OPTIMIZATION_HEADERS
   definitions.h
   details.h
   quadratic_problem.h
   integral_cost.h
   )
 
 INSTALL(FILES
-  ${${PROJECT_NAME}_HELPERS_HEADERS}
-  DESTINATION include/hpp/spline/optimization
+  ${${PROJECT_NAME}_OPTIMIZATION_HEADERS}
+  DESTINATION include/curves/optimization
```

Comment:
`${PROJECT_NAME}` would be more robust in case there is a renaming

Reply:
Done in https://github.com/loco-3d/curves/pull/20, thanks for the suggestion.